packages/manager/apps/hub/src/routing.js

```diff
@@ -107,5 +107,27 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
     },
   });
 
+  $stateProvider.state('app.userContracts', {
+    url: '/user-contracts',
+    redirectTo: ($transition$) => {
+      const { contracts } = $transition$.params();
+      return contracts ? false : 'app.dashboard';
+    },
+    views: {
+      'app@': {
+        component: 'userContracts',
+      },
+    },
+    params: {
+      contracts: null,
+    },
+    translations: { value: ['.'], format: 'json' },
```

Comment:
Should we import translations from `packages/manager/apps/hub/src/translations` for this state ?